src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -1724,7 +1724,7 @@ public IEnumerable<CompletionResult> CompleteArgument(
         private static SortedSet<string> GetCommandNouns(IDictionary fakeBoundParameters)
         {
             Collection<CommandInfo> commands = CompletionCompleters.GetCommandInfo(fakeBoundParameters, "Module", "Verb");
-            SortedSet<string> nouns = new(StringComparer.OrdinalIgnoreCase);
+            SortedSet<string> nouns = new();
```

Comment:
If cmdlet names are case-insensetive this must be case-insensetive. I think we should think about fixing tests only.

Reply:
Yep I agree, although to completer it does not make a difference. I have updated test. Only way is to use sorted set in the test because it gives different results to `Sort-Object -Unique` on Linux.